.github/workflows/bindgen.yml

```diff
@@ -0,0 +1,29 @@
+name: Check Bindgen Output
+
+on:
+  workflow_call:
+
+jobs:
+  check-bindgen:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Fetch the target branch
+        run: |
+          git fetch origin ${{ github.base_ref }}
```

Comment:
`base_ref` is `master`, right? You could just add 

```
with:
  ref: ${github.base_ref}
```

to the `actions/checkout` action

Reply:
Yeah I realized I don't actually need this 😅

Reply:
Or, if the branch is supposed to be the PR branch then it should be removed completely ofc.